src/descriptor/policy.rs

```diff
@@ -76,6 +76,19 @@ pub enum PkOrF {
     XOnlyPubkey(XOnlyPublicKey),
     /// An extended key fingerprint
     Fingerprint(Fingerprint),
+    /// serialization for xpub origin metadata
```

Comment:
Can we call this "An XPub with origin metadata"?? As per the other item docs and sounds a bit more clear to me.

Reply:
I see the confusion. Basically because there's already something called `origin` in the library (the field on the [DescriptorXKey struct](https://docs.rs/bdk/latest/bdk/descriptor/struct.DescriptorXKey.html#structfield.origin)) so it's a little confusing. I guess I'm trying to find some name that represents the full struct necessary for defining an xpub in a meaningful way (e.g. when verifying membership in a multisig wallet, which is one of the primary use cases we're building this out for). 

It's been a while since I've spent time in this code, so it's possible I'm missing something, but basically what I was originally thinking was where `XpubOrigin` is an object that contains an Extended Public Key plus it's derivation/origin metadata. 

OTOH, maybe I'm misunderstanding your comment and you're just asking to update this code comment 😅. In which case, maybe "serialization for xpub with derivation metadata" would be more clear to avoid the naming clash with origin?